packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -657,10 +678,21 @@ export default class New extends Component {
                       </div>
                     </fieldset>,
                     <fieldset>
-                      <label>{_('newSrUseNfs4')}</label>
-                      <div>
-                        <Toggle onChange={this.toggleState('nfs4')} />
-                      </div>
+                      <label htmlFor='selectSrPath'>{_('newSrUseNfs')}</label>
+                      <select
+                        className='form-control'
+                        onChange={this._handleNfsVersion}
+                        value={nfsVersion}
+                      >
+                        <option value={null}>
+                          {formatMessage(messages.newSrNfsDefaultVersion)}
```

Comment:
Default NFS is empty?  `No selected value` what do you think?

Reply:
The convention says that: 
If you don't choose NFS version --> Default NFS version will be selected automaticly by the server.
Front end side: 
If NFS verson === ' ' ---> `undefined` will be sent to the server

Reply:
Please you can leave me the link when we take into account the default NFS version (side server) because I don't see it.

Reply:
It's treated `xcp-ng` side

Reply:
IMHO, we should to differentiate the two cases(XCP-ng, XS). For `XS` users will think that's a default value but it isn't and for `XCP-ng` users don't know which is the default version.
What do you think?

Reply:
@julien-f what do you think about this ?